docker.gradle

```diff
@@ -22,14 +23,21 @@ ext {
     dockerTimeout = project.hasProperty('dockerTimeout') ? dockerTimeout.toInteger() : 840
     dockerRetries = project.hasProperty('dockerRetries') ? dockerRetries.toInteger() : 3
     dockerBinary = project.hasProperty('dockerBinary') ? [dockerBinary] : ['docker']
+    dockerBuildArgs = project.hasProperty('dockerBuildArgs') ? dockerBuildArgs : ''
 }
 
 if(project.hasProperty('dockerHost')) {
     dockerBinary += ['--host', project.dockerHost]
 }
 
 task distDocker << {
-    def cmd = dockerBinary + ['build', '-t', dockerImageName, project.buildscript.sourceFile.getParentFile().getAbsolutePath()]
+    def cmd
+    if (dockerBuildArgs != '') {
+        cmd = dockerBinary + ['build', '--build-arg', dockerBuildArgs, '-t', dockerImageName, project.buildscript.sourceFile.getParentFile().getAbsolutePath()]
+    }
+    else {
+        cmd = dockerBinary + ['build', '-t', dockerImageName, project.buildscript.sourceFile.getParentFile().getAbsolutePath()]
+    }
```

Comment:
@markusthoemmes - good idea.  I see what was done above with dockerHost, and I'll have the build args follow the same process